src/SFA.DAS.Authorization/AuthorizationService.cs

```diff
@@ -18,7 +18,8 @@ public AuthorizationService(IAuthorizationContextProvider authorizationContextPr
 
         public void Authorize(params string[] options)
         {
-            AuthorizeAsync(options).GetAwaiter().GetResult();
+            Task.Run(() => AuthorizeAsync(options)).GetAwaiter().GetResult();
+            //AuthorizeAsync(options).GetAwaiter().GetResult();
```

Comment:
If we're happy with the task.Run wrapper we could lose this commented line

Reply:
No I would like to leave the comment in, as I want to discus with elliot